sections/titlepages.tex

```diff
@@ -31,7 +31,10 @@
   9220 Aalborg East, DK\\
   \href{www.cs.aau.dk}{www.cs.aau.dk}
 }{% the abstract
-This is the best abstract ever written 
+Can Pozyx, a highly accurate positioning solution based on UWB, be used to create a virtual reality game that represents the user with real-time positional data in-game?
+In this project, we look into how to obtain positional data, model a network protocol for transferring the data to the players, and create a virtual reality-based game that shows the data.
+In developing the project, the pragmatic paradigm presented by Ivan Aaen in his research about Essence will be utilized to encourage innovation and diverse thinking in the development process.
+The results show that while Pozyx provides high accuracy, the refresh rate that it provides is not high enough for a real-time experience.
```

Comment:
er the refresh rate we achieved ikke mere korrekt?

Reply:
```suggestion
The results show that while Pozyx provides high accuracy, the refresh rate that we were able to achieve using it was not fast enough for a real-time experience.
```